internal/policy/errors.go

```diff
@@ -0,0 +1,19 @@
+package policy
+
+import "fmt"
+
+type PolicyLoaderError struct {
+	loaderError error
+}
+
+func (e *PolicyLoaderError) Error() string {
```

Comment:
I think it's better that the errors names are like `ErrSomethingWentWrong`, like some (if not all) packages from the stdlib do. It's easier to look for them IMHO. What do you think?

```suggestion
type ErrPolicyLoad struct {
	loaderError error
}

func (e *ErrPolicyLoad) Error() string {
```

Reply:
Still picking up these standards. Will change!